README.md

```diff
@@ -10,6 +10,9 @@ This code is licensed under the terms of the GNU Affero General Public License v
 
 * Install [the latest release](https://github.com/arduino/arduino-cloud-cli/releases).
 * A client ID and a secret ID, retrievable from the [cloud](https://cloud.arduino.cc/home/api-keys) by creating a new API key.
+* Optional: the command `arduino-cloud-cli device create` requires the `arduino-cli` tool installed on the system. 
+  So, if you plan to create a device on the Arduino Cloud via CLI, you need to install the `arduino-cli` following the instructions at [https://arduino.github.io/arduino-cli/0.32/](https://arduino.github.io/arduino-cli/0.32/)
```

Comment:
```suggestion
  So, if you plan to create a device on the Arduino Cloud via CLI, you need to install the `arduino-cli` following the instructions at [https://arduino.github.io/arduino-cli/latest/installation/](https://arduino.github.io/arduino-cli/latest/installation/)
```

- Use `latest` in the URL path to automatically redirect to the documentation for the latest release of Arduino CLI instead of pinning to the 0.32.x-specific docs
- Link directly to the installation instructions

Reply:
Thanks @per1234 for your suggestion, it totally makes sense.